test/pg-cdc/types-unsupported.td

```diff
@@ -28,4 +28,4 @@ CREATE PUBLICATION mz_source FOR ALL TABLES;
 
 # Todo: add test for detail and hint once testdrive supports that
 ! CREATE SOURCE mz_source FROM POSTGRES CONNECTION pg (PUBLICATION 'mz_source') FOR ALL TABLES;
-contains: column "person.current_mood" uses unrecognized type
+regex: the following columns contain unsupported types:\npostgres\.public\.person\.current_mood \(OID \d+\)
```

Comment:
This is now possible in testdrive, see recently merged PR:

https://github.com/MaterializeInc/materialize/pull/16030/files#diff-0e75185705f0683fc07a43771efee7c26becb34e9b7d5e82db62e29619f9d284

Reply:
I mentioned this because of this comment: `Todo: add test for detail and hint once testdrive supports that`

Reply:
Oh sorry--I totally overlooked the `Todo`; ty